.github/workflows/linting-and-tests.yml

```diff
@@ -69,9 +69,9 @@ jobs:
         if: steps.cache-frontend-dependencies.outputs.cache-hit != 'true'
         working-directory: grafana-plugin
         run: yarn install --frozen-lockfile --prefer-offline --network-timeout 500000
-      - name: Build frontend (will run linter and tests)
+      - name: Build, lint and test frontend
         working-directory: grafana-plugin
-        run: yarn build
+        run: yarn lint && yarn test && yarn build
```

Comment:
previously `yarn build` used to run the linting and tests, I guess w/ the move to the new plugin toolkit build tool version that was no longer the case?